.github/workflows/ci.yml

```
@@ -160,7 +160,7 @@ jobs:
       #   run: ./test/Scripts.Integration.Test/build-project.ps1 -UnityPath "${{ env.UNITY_PATH }}" -Platform "${{ matrix.platform }}"
```

```suggestion
      #   run: ./test/Scripts.Integration.Test/build-project.ps1 -UnityPath "${env:UNITY_PATH}" -Platform "${{ matrix.platform }}"
```
Just in case someone uncomments this.